packages/react/src/Testimonial/Testimonial.tsx

```diff
@@ -89,14 +113,34 @@ const Root = forwardRef(_Root)
  * <Testimonial.Quote>
  */
 type QuoteProps = {
-  children: string
+  children: string | Array<string | React.ReactElement<HTMLElement>>
 } & React.HTMLAttributes<HTMLElement> &
   BaseProps<HTMLElement>
 
 function _Quote({children, className}: QuoteProps, ref) {
+  const childrenArray = useMemo(() => React.Children.toArray(children), [children])
+
+  // TODO: when Firefox supports :has() selector, we should use that instead of JS
+  const getConditionalVariant = useCallback(() => {
```

Comment:
Assuming this merges after the river breakout, would be great to replace this with the helper component you introduced over there.